mesonbuild/compilers/mixins/clang.py

```diff
@@ -165,6 +165,12 @@ def get_lto_compile_args(self, *, threads: int = 0, mode: str = 'default') -> T.
             args.extend(super().get_lto_compile_args(threads=threads))
         return args
 
+    def linker_to_compiler_args(self, args: T.List[str]) -> T.List[str]:
+        if isinstance(self.linker, (ClangClDynamicLinker, MSVCDynamicLinker)):
+            return list(map(lambda flag: flag if flag.startswith('-Wl,') else f'-Wl,{flag}', args))
```

Comment:
Nitpick: this can just be a list comprehension:

```python
return [flag if flag.startswith('-Wl,') else f'-Wl,{flag}' for flag in args]
```

Much simpler to understand than list() + map() + lambda.

Reply:
Thanks!